crates/client/block_import/src/types.rs

```diff
@@ -142,13 +142,14 @@ pub struct UnverifiedCommitments {
 }
 
 /// An unverified pending full block as input for the block import pipeline.
-#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, Eq, PartialEq, Default, Serialize, Deserialize)]
 pub struct UnverifiedPendingFullBlock {
     pub header: UnverifiedHeader,
     pub state_diff: StateDiff,
     pub transactions: Vec<Transaction>,
     pub receipts: Vec<TransactionReceipt>,
     pub declared_classes: Vec<DeclaredClass>,
+    pub visited_segments: Option<Vec<(Felt, Vec<usize>)>>,
```

Comment:
Can we have a struct so that the types are better described

Reply:
maybe just add documentation